src/site/content/en/blog/compat2021-midyear/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Compat2021 mid-year update: Flex gap everywhere"
+description:
+    "Mid-year update on Compat2021 —an effort to eliminate browser compatibility problems in five 
```

Comment:
```suggestion
    "Mid-year update on Compat 2021 —an effort to eliminate browser compatibility problems in five 
```

Reply:
I see there's also inconsistent space around "—" here. I'll suggest removing since this is an em dash.